tests/scripts/all.sh

```diff
@@ -866,19 +866,18 @@ component_check_test_cases () {
     fi
     tests/scripts/check_test_cases.py $opt
     unset opt
+
+    # Check if some test are explicitely disabled when USE_PSA_CRYPTO is set
+    not grep -n -R 'depends_on.*!MBEDTLS_USE_PSA_CRYPTO' tests/suites/*.function tests/suites/*.data
+    not grep -n -R '^ *requires_config_disabled.*MBEDTLS_USE_PSA_CRYPTO' tests/ssl-opt.sh tests/opt-testcases/*.sh
```

Comment:
https://github.com/Mbed-TLS/mbedtls/pull/5742#discussion_r855110959 is not addressed.

Reply:
Indeed, I mixed with other comments, but this one was better.